extension/js/common/composer.ts

```diff
@@ -302,6 +304,17 @@ export class Composer {
     }, this.getErrHandlers('delete draft')));
     this.S.cached('body').bind({ drop: Ui.event.stop(), dragover: Ui.event.stop() }); // prevents files dropped out of the intended drop area to screw up the page
     this.S.cached('icon_sign').click(Ui.event.handle(() => this.toggleSignIcon(), this.getErrHandlers(`enable/disable signing`)));
+    $(window).resize(() => {
```

Comment:
I added `$(window).resize(Ui.event.prevent('slowspree', () => {` but the lag is visible, please check it when you review the code again.

Reply:
Thanks, I will check

Reply:
It is indeed quite noticeable. I ended up changing the event delay based on it being a new message vs a reply. See https://github.com/FlowCrypt/flowcrypt-browser/pull/1728/commits/79b6b59eb202506ddcc5d5a98c9cfbc37daffe6f let me know if it looks ok to you.

Reply:
Yes, it looks great.
I didn't understand why we should use `Ui.event.prevent('slowspree'` but now I realized that users can just manually resize the whole browser window and it will be triggered.
Thanks!

Reply:
Yes these things can be tricky. I would not have thought of that either, I added the delays as a result of debugging user issues.